registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/RegistrationStatusService.java

```diff
@@ -1,141 +1,141 @@
-	public U getRegistrationStatus(String regid, String processs, Integer iteration, String workflowInstanceId);
-
-	public List<InternalRegistrationStatusDto> getAllRegistrationStatuses(String registrationId);
-
-	public InternalRegistrationStatusDto getRegStatusForMainProcess(String registrationId);
-
-	/**
-	 * Adds the registration status.
-	 *
-	 * @param registrationStatusDto
-	 *            the registration status dto
-	 * @param moduleId
-	 *            the module id
-	 * @param moduleName
-	 *            the module name
-	 */
-	public void addRegistrationStatus(U registrationStatusDto, String moduleId, String moduleName);
-
-	/**
-	 * Update registration status.
-	 *
-	 * @param registrationStatusDto
-	 *            the registration status dto
-	 * @param moduleId
-	 *            the module id
-	 * @param moduleName
-	 *            the module name
-	 */
-	public void updateRegistrationStatus(U registrationStatusDto, String moduleId, String moduleName);
-
-	/**
-	 * Gets the by status.
-	 *
-	 * @param status
-	 *            the status
-	 * @return the by status
-	 */
-	public List<U> getByStatus(String status);
-
-	/**
-	 * Gets the by ids.
-	 *
-	 * @param requestIds
-	 *            the request ids
-	 * @return the list of Registrations for the given ids.
-	 */
-	public List<D> getByIds(List<RegistrationStatusSubRequestDto> requestIds);
-
-	/**
-	 * Gets the un processed packets.
-	 *
-	 * @param fetchSize
-	 *            the fetch size
-	 * @param elapseTime
-	 *            the elapse time
-	 * @param reprocessCount
-	 *            the reprocess count
-	 * @param status
-	 *            the status
-	 * @return the un processed packets
-	 */
-	public List<U> getUnProcessedPackets(Integer fetchSize, long elapseTime, Integer reprocessCount,
-			List<String> status);
-
-	/**
-	 * Gets the un processed packets count.
-	 *
-	 * @param elapseTime
-	 *            the elapse time
-	 * @param reprocessCount
-	 *            the reprocess count
-	 * @param status
-	 *            the status
-	 * @return the un processed packets count
-	 */
-	public Integer getUnProcessedPacketsCount(long elapseTime, Integer reprocessCount, List<String> status);
-
-	/**
-	 * Check Rid if uin is available.
-	 *
-	 * @param rid
-	 *            the rid
-	 * @return the boolean
-	 */
-	public Boolean checkUinAvailabilityForRid(String rid);
-
-	/**
-	 * Gets the by ids and timestamp.
-	 *
-	 * @param ids
-	 *            the ids
-	 * @return the by ids and timestamp
-	 */
-	public List<U> getByIdsAndTimestamp(List<String> ids);
-
-	public List<InternalRegistrationStatusDto> getActionablePausedPackets(Integer fetchSize);
-
-	public Page<InternalRegistrationStatusDto> searchRegistrationDetails(SearchInfo searchInfo);
-
-	public void updateRegistrationStatusForWorkflow(U registrationStatusDto, String moduleId, String moduleName);
-
-	public List<InternalRegistrationStatusDto> getResumablePackets(Integer fetchSize);
-
-}
+package io.mosip.registration.processor.status.service;

```

Comment:
If there is no changes in this file, please don't commit

Reply:
This is already committed some space changes only